platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerTest.java

```diff
@@ -618,14 +618,275 @@ public void testDoImportRecordsMigrationExceptionWithInvalidChecksum() throws Ex
                 + ".dar]."));
   }
 
-  private void expectImportDelegationIsSuccessful()
-      throws NoSuchPaddingException, NoSuchAlgorithmException {
+  @Test
+  public void doDecryptSucceeds() throws Exception {
+    final Path path = ddfHome.resolve(TEST_DIRECTORY);
+
+    configurationMigrationManager = spy(getConfigurationMigrationManager());
+
+    // Need to actually generate a zip with a valid checksum so calling export
+    configurationMigrationManager.doExport(path);
+
+    expectImportDelegationIsSuccessful();
+
+    final MigrationReport report = configurationMigrationManager.doDecrypt(path);
+
+    assertThat("Decrypt was not successful", report.wasSuccessful(), is(true));
+    reportHasInfoMessage(
+        report.infos(),
+        equalTo(
+            "Successfully decrypted file ["
+                + path.resolve(TEST_BRANDING)
+                + "-"
+                + TEST_VERSION
+                + ".dar] to ["
+                + path.resolve(TEST_BRANDING)
+                + "-"
+                + TEST_VERSION
+                + ".zip]."));
+    verify(configurationMigrationManager)
+        .delegateToDecryptMigrationManager(
+            any(MigrationReportImpl.class), any(MigrationZipFile.class), any(Path.class));
```

Comment:
❓ Should we at least verify that the right `Path` was used and not just any path?

Reply:
done